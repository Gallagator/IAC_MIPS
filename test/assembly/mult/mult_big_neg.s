	.text
	.align	2
	.globl	main
	.ent	main
main:
	.set	noreorder
    lui $3, 0xA463
    lui $4, 0x6748
    addiu $4, $4, 0x45
    mult $3, $4
    mfhi $2
    jr $0
    mflo $4
.end	main
