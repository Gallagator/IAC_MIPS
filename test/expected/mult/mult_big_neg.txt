0xdb0a10bf